packages/drivers/odsp-driver/src/odspUtils2.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { fetchTokenErrorCode, throwOdspNetworkError } from "@fluidframework/odsp-doclib-utils";
+import {
+    IOdspResolvedUrl,
+    isTokenFromCache,
+    OdspResourceTokenFetchOptions,
+    TokenFetcher,
+    TokenFetchOptions,
+    tokenFromResponse,
+} from "@fluidframework/odsp-driver-definitions";
+import { PerformanceEvent } from "@fluidframework/telemetry-utils";
+import { IOdspSnapshot } from "./contracts";
+
+export function evalBlobsAndTrees(snapshot: IOdspSnapshot) {
```

Comment:
Rather than an `odspUtils2.ts`, what about just creating two separate files: `evalBlobsAndTrees.ts` and `toInstrementedOdspTokenFetcher.ts`?

Reply:
I think I will rename the file to prefetchSnapshotUtils.ts and add a comment not to add general utils to that file instead of creating new files for every new function.

Reply:
Actually I added these to odsputils only because we already import a lot from that file. So no need to add these in a separate file.